File: src/main/java/ru/iamdvz/magik/utils/ItemsUtil.java
package ru.iamdvz.magik.utils;

import org.bukkit.inventory.ItemStack;
import org.bukkit.inventory.meta.ItemMeta;

public class ItemsUtil {

    public static ItemStack setCMD(ItemStack item, int CMD) {
        ItemMeta itemMeta = item.getItemMeta();
        itemMeta.setCustomModelData(CMD);
        item.setItemMeta((itemMeta));
        return item;
    }
}

File: src/main/java/ru/iamdvz/magik/utils/ItemsUtil.kt
package ru.iamdvz.magik.utils

import org.bukkit.inventory.ItemStack
import org.bukkit.inventory.meta.ItemMeta

object ItemsUtil {
    @JvmStatic
    fun setCmd(item: ItemStack, CMD: Int?): ItemStack {
        val itemMeta: ItemMeta = item.itemMeta
        itemMeta.setCustomModelData(CMD)
        item.setItemMeta(itemMeta)
        return item
    }
}